.github/workflows/windows-compile.yml

```diff
@@ -0,0 +1,35 @@
+# Platform specific Windows test. See verible-ci for main continuous integration.
+name: windows-compile
+
+on:
+  push:
+    branches:
+    - master
+    - main
+  pull_request:
+    branches:
+    - master
+    - main
+
+jobs:
+  WindowsBuild:
+    runs-on: windows-latest
+    steps:
+    # Need winflexbison for Kythe, bazel fails if environment
+    # FLEX/BISON is not set to a valid program
+    - name: Install deps
+      run: |
+        choco install winflexbison
+    - name: Checkout code
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    # Only check flex/bison targets at the moment, since Verible cannot be
+    # fully compiled on Windows (yet!)
+    - name: Build flex/bison files
+      run: |
+        $env:FLEX = 'C:/ProgramData/Chocolatey/lib/winflexbison/tools/win_flex.exe'
```

Comment:
This sounds oddly specific and I am worried that this is brittle and might break easily.

Is there a way to find the path where the binaries end up in some other way ? I could imagine that the `C:/ProgramData/Chocolatey/` is the base-path where `choco` stores things, maybe it can be queried what its base-path is, so that we can assemble the final path from there ?

(On Mac-OS for instance, we ask the `brew` package manager [where it installed binaries]( https://github.com/google/verible/blob/6b827debc9dbd7da2d1a4adad8a16113e0083ac0/.github/workflows/macos-compile.yml#L27) )

Reply:
Question for my understanding: the rules don't actually build flex and bison in the hermetic build from source (like they do on the Unixes), but rely on the chocolatey system-installed ones, this is why we need this in the first place ?

(Similar to what Kythe is doing for its building, which requires us to install flex and bison for the Unixy builds currently)

Reply:
For Verible, the rules I added download the github released binary and uses that in the genlex/genyacc. There is no need for a system installed flex/yacc.

However, the WORKSPACE calls [lexyacc_configure from Kythe](https://github.com/kythe/kythe/blob/160b28af24a477fd75458e033523cf9ce81733ee/tools/build_rules/lexyacc/lexyacc.bzl#L156), which expects to find flex/bison on the path or in the environment variables FLEX/BISON, so a valid flex/bison must be installed to satisfy Kythe (otherwise, bazel fails instantaneously). On Windows, we have to use the environment variable because the name is different than what Kythe expects (win_flex.exe vs flex...).

When building the Kythe's lex/yacc targets, the system installed flex/bison gets used (same as Unix).

There is now a way to [forgo installing flex/bison](https://github.com/kythe/kythe/pull/4880) on the system and use a bazel label instead. If you want to update to latest Kythe, I could provide a pull-request for that, which remove usage of the system flex/bison.

Reply:
Ok cool. We are soon actually not needing Kythe direct dependency anymore (one step in #772) , so we might be able to reduce the dependency soon here as well.